WebsiteChallenge-eemeelyo/css/style.css

```
@@ -0,0 +1,346 @@
+* {                               /* RESET-------------*/
+  margin: 0;
+  padding: 0;
+  outline: none;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style-type: none;
+  font-family: 'Roboto', sans-serif;
+
+}
+
+/*MAIN ------------------------------------------------------------------------MAIN */
+html, body {
+  line-height: 1.7em;
+  background-color:#caffbf;
+}
+
+a {
+  color:greenyellow;
+  text-decoration: none;
+}
+
+h1, h2, h3 {
+  padding-bottom: 20px;
+}
+
+p {
+  margin: 10px 0;
+}
+
+/*-UTLITIES --------------------------------------------------------------------UTILITIES */
+.container {
+  margin: auto;
+  /* width: 1200px; */
+}
+
+.highlight {
+ color: #ff9b85;
+ font-family: 'Zilla Slab Highlight', cursive;
+}
+
+.highlight2 {
+  color: #caffbf;
+  font-family: 'Zilla Slab Highlight', cursive;
+ }
+
+.highlight-title {
+ color: #ff9b85;
+ font-family: 'Allura', cursive;
+ font-size: 70px;
+ margin: 80px 50px 50px 50px;
+}
+
+.cursive {
+  font-family: 'Allura', cursive;
+}
+
+.archi {
+  font-family: 'Architects Daughter', cursive;
+  color: #f566a3;
+}
+
+.roboto {
+  font-family: 'Roboto', sans-serif;
+}
+
+.clr {
+  clear: both;
+}
+
+.bg-primary {
+  background: #f566a3;
+  color: #caffbf;
+}
+
+/* NAVBAR ---------------------------------------------------------------------- NAVBAR */
+nav {
+  font-size: 2rem;
+}
+.nav-list{
+  display: flex;
+  justify-content: flex-end;
+  padding: 0 2rem;
+  
+}
```

Try using prettier to ensure that code formatting is consistent.  Throughout all of your code.

```suggestion
nav {
  font-size: 2rem;
}

.nav-list {
  display: flex;
  justify-content: flex-end;
  padding: 0 2rem;
}
```